docs/source/introduction/quickstart.ipynb

```diff
@@ -13,7 +13,16 @@
    "cell_type": "code",
```

Comment:
<ul><li>It is known that encoding prior knowledge leads to faster <strong>convergence</strong> to the true solution...</li><li>help find the solution to the current linear <strong>system</strong> faster...</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probabilistic-numerics/probnum/pull/419/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
done